examples/pytorch/CaptumExample/README.md

```diff
@@ -0,0 +1,61 @@
+## Titanic Basic data analysis using Captum and mlflow. 
+In this example, we will demonstrate the basic features of the Captum interpretability,and loging those features using mlflow library through an example model trained on the Titanic survival data. 
+We will first train a deep neural network on the data using PyTorch and use Captum to understand which of the features were most important and how the network reached its prediction.
+
+### Running the code
+To run the example via MLflow, navigate to the `mlflow/examples/pytorch/CaptumExample` directory and run the command
+
+```
+mlflow run .
+```
+
+This will run `Titanic_Captum_Interpret.py` with the default set of parameters such as  `--max_epochs=100` and `--use_pretrained_model False`. You can see the default value in the `MLproject` file.
```

Comment:
```suggestion
This will run `Titanic_Captum_Interpret.py` with default parameter values, e.g.  `--max_epochs=100` and `--use_pretrained_model False`. You can see the full set of parameters in the `MLproject` file within this directory.
```